### PR TITLE
Initial minutes template 2023-APR-20

### PR DESCRIPTION
**Pull request by sthagen**
Please kindly review the minutes of meeting 2023-APR-20

### CODE REVIEW

**Review by dmk42 — APPROVED** [review]
Thanks, Stefan!

**Review by katrinaoneil — APPROVED** [review]

File: meeting_minutes/230420_SARIF_TC_74.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+* Pavel: What is the status of supporting dynamic (DAST) and interactive (IAST) tools?
+* David K.: Walks all through the relevant section in the document provided by Jeff per [the pull request "Create Future.md" #560](https://github.com/oasis-tcs/sarif-spec/pull/560)
+
+# 6. Resolutions and Decisions reached (by 10 minutes prior to scheduled meeting end)
+
+## 6.1 End debate of other issues by 10 minutes prior to scheduled meeting end and follow the agenda from this point (Co-Chair David)
+
+## 6.2 Review of Decisions Reached (Secretary Stefan)
+
+* DECISION to meet next on May 25, 2023
```
Comment (pavel-mikula-sonarsource) [inline meeting_minutes/230420_SARIF_TC_74.md:124]: Wasn't there also the decision to meet on June 15?

Comment (dmk42) [reply meeting_minutes/230420_SARIF_TC_74.md:124]: Good catch.

Comment (sthagen) [reply meeting_minutes/230420_SARIF_TC_74.md:124]: I suggest to leave as is. We often do mention in decisions only the next meeting. During the meeting I edit the proposed meetings in the minutes to match the outcome of the discussion. Now we already note May 25 three times: In the proposed section, in the decisions, and in the next meeting section.

**Pull request merged**